docs/types.rst

```diff
@@ -862,15 +875,34 @@ Members
         // dynamic array of pairs (i.e. of fixed size arrays of length two).
         // Because of that, T[] is always a dynamic array of T, even if T
         // itself is an array.
+        // Data location for all state variables is storage.
         bool[2][] m_pairsOfFlags;
 
         // newPairs is stored in memory - the only possibility
-        // for public function arguments
+        // for public contract function arguments
         function setAllFlagPairs(bool[2][] memory newPairs) public {
-            // assignment to a storage array replaces the complete array
+            // assignment to a storage array performs a copy of ``newPairs`` and
+            // replaces the complete array ``m_pairsOfFlags``.
             m_pairsOfFlags = newPairs;
         }
 
+        struct StructType {
+            uint[] contents;
+            uint moreInfo;
+        }
+        StructType s;
+
+        function f(uint[] memory c) public {
```

Comment:
Is this example still compiling with 0.4.16 (that is the pragma above) ?

Reply:
confirmed.